### ISSUE TITLE
Error after latest update - TypeError: DockManager.getDefault() is null

### ISSUE THREAD

**Issue by neuromorph** [micheleg/dash-to-dock#2269]
After latest update of the extension, `journalctl` is getting flooded with error:

```
fedora gnome-shell[3353]: JS ERROR: TypeError: DockManager.getDefault() is null
                                             get settings@file:///home/sergey/.local/share/gnome-shell/extensions/dash-to-dock@micxgx.gmail.com/docking.js:1752:28
                                             updateNotificationBadgeStyle@file:///home/sergey/.local/share/gnome-shell/extensions/dash-to-dock@micxgx.gmail.com/appIconIndicators.js:788:33
                                             _unloadExtensionStylesheet@resource:///org/gnome/shell/ui/extensionSystem.js:157:15
                                             _callExtensionDisable@resource:///org/gnome/shell/ui/extensionSystem.js:222:14
                                             _onEnabledExtensionsChanged@resource:///org/gnome/shell/ui/extensionSystem.js:639:24
                                             async*_sessionUpdated@resource:///org/gnome/shell/ui/extensionSystem.js:826:20
                                             ExtensionManager/<@resource:///org/gnome/shell/ui/extensionSystem.js:48:18
                                             _callHandlers@resource:///org/gnome/gjs/modules/core/_signals.js:130:42
                                             _emit@resource:///org/gnome/gjs/modules/core/_signals.js:119:10
                                             _sync@resource:///org/gnome/shell/ui/sessionMode.js:207:14
                                             pushMode@resource:///org/gnome/shell/ui/sessionMode.js:169:14
                                             activate@resource:///org/gnome/shell/ui/screenShield.js:616:34
                                             lock@resource:///org/gnome/shell/ui/screenShield.js:683:14
                                             LockAsync@resource:///org/gnome/shell/ui/shellDBus.js:525:28
                                             _handleMethodCall@resource:///org/gnome/gjs/modules/core/overrides/Gio.js:373:35
                                             _wrapJSObject/<@resource:///org/gnome/gjs/modules/core/overrides/Gio.js:408:34
                                             @resource:///org/gnome/shell/ui/init.js:21:20
```

Hope this helps to look into it. I am not on the system now but noted this from another user and didn't see similar error mentioned in issues.
Thanks.

**Comment by JoaquimCarneiro** [comment]
I'm getting the same issue. And enabling the extension causes lots of stutters

**Comment by XMoon** [comment]
same here, must reboot the system to fix the stutters.

gnome 46.4 with dash-to-dock v93

temporarily change `appIconIndicators.js:788` to an fixed value fix it.

**Comment by tareqimbasher** [comment]
I'm getting alot of stutters too once I upgraded to v94. System hangs for 1 second, every 10 seconds or so.

Arch Linux
Gnome 46.4

Disabling the extension makes the stutters go away only after rebooting.

**Comment by 3v1n0** [comment]
Duplicate of #2270
